scripts/setup.py

```diff
@@ -0,0 +1,86 @@
+import sys
+import os
+
+import click
+from jinja2 import Environment, FileSystemLoader
+import secrets
+
+FILE_PATH = os.path.dirname(os.path.abspath(__file__))
+PROJECT_PATH = os.path.abspath(os.path.join(FILE_PATH, os.path.pardir))
+GROUP_VARS_PATH = 'deployment/ansible/group_vars'
+GROUP_VARS_TEMPLATE_FILENAME = 'production.j2'
+GROUP_VARS_FILENAME = 'production'
+INVENTORY_PATH = 'deployment/ansible/inventory'
+INVENTORY_TEMPLATE_FILENAME = 'production.j2'
+INVENTORY_FILENAME = 'production'
+
+
+def render_template(template_path, template_filename, context):
+    template_environment = Environment(
+        autoescape=False,
+        loader=FileSystemLoader(template_path),
+        trim_blocks=False)
+    return template_environment.get_template(template_filename).render(context)
+
+
+def generate_password():
+    return secrets.token_urlsafe(32)
+
+
+def wrap_in_quotes(s):
+    return '"{}"'.format(s)
+
+
+@click.command()
+@click.option(
+    '--app_domain_name',
+    prompt='App domain name (without http:// or https://)',
+)
+@click.option(
+    '--database_server_ip',
+    prompt='Database server IP',
```

Comment:
I think these should be clear that these three  are the _internal_ IPs, which the servers will use to talk to one another; they may not necessarily be the same as the IPs used in the inventory file.